src/_data/platforms.yml

```diff
@@ -1,50 +1,50 @@
 - platform: 'Android SDK'
   chipsets: 'x64, Arm32, Arm64'
   supported: '21 to 34'
-  besteffort: 'N/A'
+  ci-tested: '21 to 34'
   unsupported: '20 and earlier'
 - platform: 'iOS'
   chipsets: 'Arm64'
-  supported: '17'
-  besteffort: '12 to 16, 18'
+  supported: '12 to 18'
+  ci-tested: '16, 17'
   unsupported: '11 and earlier'
 - platform: 'macOS'
   chipsets: 'x64, Arm64'
-  supported: '<abbr title="Ventura">13</abbr>, <abbr title="Sonoma">14</abbr>'
-  besteffort: '<abbr title="Mojave">10.14</abbr> to <abbr title="Monterey">12</abbr>, <abbr title="Sequoia">15</abbr>'
-  unsupported: '<abbr title="High Sierra">10.13</abbr> and earlier'
+  supported: 'Mojave (10.14) to Sequoia (15)'
+  ci-tested: 'Ventura (13), Sonoma (14)'
+  unsupported: 'High Sierra (10.13) and earlier'
 - platform: 'Windows'
   chipsets: 'x64, Arm64'
-  supported: '10'
-  besteffort: '11'
+  supported: '10, 11'
+  ci-tested: '10'
   unsupported: '8 and earlier'
 - platform: 'Debian (Linux)'
   chipsets: 'x64, Arm64'
-  supported: '11, 12'
-  besteffort: '10'
+  supported: '10, 11, 12'
+  ci-tested: '11, 12'
   unsupported: '9 and earlier'
 - platform: 'Ubuntu (Linux)'
   chipsets: 'x64, Arm64'
   supported: '20.04 LTS, 22.04 LTS'
-  besteffort: '24.04 LTS'
+  ci-tested: '22.04 LTS'
   unsupported: '23.10 and earlier non-LTS'
 - platform: 'Chrome (Web)'
   chipsets: 'N/A'
   supported: '[Latest 2](https://chromereleases.googleblog.com/search/label/Stable%20updates)'
```

Comment:
I think this should be `96 until Latest 2`. Same comment for other browsers. @yjbanov to confirm.

Reply:
Are we changing the structure of the table? What we currently have in https://docs.flutter.dev/reference/supported-platforms LGTM.

Reply:
We're getting rid of "best effort" and replacing it with "CI-tested versions" (which of course has different values in it)

Reply:
@zanderso just explained to me how the table is changing. So for web we'll want this:

- Chrome and Edge:
  - Supported: Latest 2 versions
  - CI-tested: 119, 125
- Firefox:
  - Supported: 106 and newer
  - CI-tested: 106
- Safari. I'm still confirming the exact CI version we're using, but we _want_ it to be this:
  - Supported: 16.6 and newer
  - CI-tested: 16.6